metapool/metapool.py

```diff
@@ -44,6 +46,19 @@ def read_plate_map_csv(f, sep='\t'):
         # still indexed with a continuous list of integers
         plate_df = plate_df[~null_samples]
         plate_df.reset_index(inplace=True, drop=True)
+    
+    try: #Check that there are no duplicated sample names, if there are duplicates then throws an assertion error and doesn't create plates_df
+        assert(len(set(plate_df['Sample'])) == len(plate_df['Sample']))
+    except AssertionError as e:
+        prev = ''
+        for sample in sorted(plate_df['Sample']):
+            if sample == prev:
+                print('\nDuplicates:')
+                print(plate_df.loc[plate_df['Sample'] == prev,])
+
+            prev = sample
+        print('\n\nERROR! Some samples names are duplicate! Please update plate map to fix duplciates')
+        raise e
```

Comment:
This makes sense, but I think we can use Pandas' [`duplicated`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.duplicated.html) method to do this in a more succinct way:

```suggestion
    duplicated_samples = plate_df.Sample[plate_df.Sample.duplicated()]
    if len(duplicated_samples):
        raise ValueError('The following sample names are duplicated %s' %
                         ', '.join(sorted(duplicated_samples)))
```

I also changed the exception to a ValueError exception, if these changes seem good to you, then you'll need to update the docstring and the test to match the new exception class.